src/v/cluster/commands.h

```diff
@@ -194,6 +197,12 @@ using cluster_config_status_cmd = controller_command<
   cluster_config_status_cmd_type,
   model::record_batch_type::cluster_config_cmd>;
 
+using feature_update_cmd = controller_command<
+  feature_update_cmd_data,
+  int8_t, // unused
```

Comment:
The idea of a key was to enable compaction in controller partition, but since it wasn't actually taken into account in all the commands we will not be able to do it